alphascreener/monitoring/sampler.py

```
@@ -0,0 +1,463 @@
+        _alert_severity(session_factory, task_id, "critical", "oom_kill", float(oom_count))
+        return "critical"
+    return None
+
+
+def _read_oom_kill_count(pid: int) -> int:
+    try:
+        oom_path = f"/proc/{pid}/oom_score"
+        if not os.path.exists(oom_path):
+            return 0
+        return 0
+    except Exception:
```

Ruff (0.15.13)

[warning] Do not catch blind exception: `Exception`

(BLE001)